lib/foreman_tasks_core/task_launcher/group.rb

```diff
@@ -0,0 +1,122 @@
+require 'foreman_tasks_core/runner'
+
+module ForemanTasksCore
+  module TaskLauncher
+    class AbstractGroup < Batch
+      class GroupParentAction < ForemanTasksCore::TaskLauncher::ParentAction
+        def plan(launcher, input_hash)
+          launcher.launch_children(self, input_hash)
+          sequence do
+            results = plan_self
+            plan_action BatchCallback, launcher.prepare_batch(input_hash), results.output[:results]
+          end
+        end
+
+        def run(event = nil)
+          super unless event == Dynflow::Action::Skip
+        end
+
+        def check_for_errors!(optional = true)
+          super unless optional
+        end
+
+        def on_finish
```

Comment:
Where is this method called from?

Reply:
https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/action/with_sub_plans.rb#L137